state-synchronizer/src/request_manager.rs

```diff
@@ -439,7 +447,13 @@ impl RequestManager {
                 self.multicast_level + 1,
                 self.upstream_config.upstream_count(),
             );
-            debug!("[state sync] multicast timeout occurred for version {}, update multicast level from {} to {}", version, prev_multicast_level, self.multicast_level);
+            debug!(
```

Comment:
should this be an info? probably should just open up state-sync and write comments there...